### PR TITLE
Update D:\Git\winget-pkgs\manifests\m\mIRC\mIRC\7.69

### PR DESCRIPTION
**Pull request by Trenly**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114491&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/mIRC/mIRC/7.69/mIRC.mIRC.installer.yaml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
 InstallerSuccessCodes:
 - 1223
 Installers:
 - Architecture: x86
-  InstallerType: nullsoft
   InstallerUrl: https://www.mirc.com/downloads/mirc/37/mirc769.exe
-  InstallerSha256: B642E88F695897E97932633F852AC9BFD67A19A0C676C166351402A9AAFB8C6D
+  InstallerSha256: B04292AE7356A133FD64B99807DD5352792F027DB54209A79EF76F5415816421
```
Comment (stephengillie) [inline manifests/m/mIRC/mIRC/7.69/mIRC.mIRC.installer.yaml:13]:
```suggestion
  InstallerSha256: 4358DF45209E9703EC378D41BE04792290E3D7AE7E1AE7F71C31C2A0681F19E3
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**